src/cmark.h

```diff
@@ -86,6 +86,8 @@ typedef struct cmark_node cmark_node;
 typedef struct cmark_parser cmark_parser;
 typedef struct cmark_iter cmark_iter;
 
+typedef void (*CMarkNodeUserDataFreeFunc) (void *user_data);
```

Comment:
As you like, I'm used to camel case for such types but if that's not the convention I'll change it